diccionari-multilingue.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Template Name: Diccionari Multilingüe
+ *
+ * @package wp-softcatala
+ */
+
+/* JS scripts */
+wp_enqueue_script( 'sc-js-typeahead', get_template_directory_uri() . '/static/js/typeahead.js', array('sc-js-main'), '1.0.0', true );
+wp_enqueue_script( 'sc-js-multilingue', get_template_directory_uri() . '/static/js/multilingue.js', array('sc-js-typeahead'), '1.0.0', true );
+wp_localize_script( 'sc-js-multilingue', 'scajax', array(
+    'ajax_url' => admin_url( 'admin-ajax.php' )
+));
+
+$url_api = 'https://www.softcatala.org/diccionari-multilingue/api/';
+
+$context = Timber::get_context();
+$post = new TimberPost();
+$context['post'] = $post;
+$context['paraula'] = urldecode( get_query_var('paraula') );
+$context['lletra'] = urldecode( get_query_var('lletra') );
+
+if( ! empty ( $context['paraula'] ) ) {
+    $url = $url_api.'search/' . $context['paraula'];
+    if ( get_query_var('llengua') ) {
+        $lang = urldecode( get_query_var('llengua') );
+        $url = $url . '?lang='. $lang;
+        $context['lang'] = $lang;
+    }
+    $api_response = json_decode( file_get_contents( $url ) );
+
+    if ( isset( $api_response[0] ) ) {
+        $response['result'] = $api_response[0];
+    } else {
+        $response['message'] = 'Sembla que la paraula que esteu cercant no es troba al diccionari. Heu seleccionat la llengua correcta?';
```

Comment:
No estic segur que la millor opció siga un 404. Pot ser algú està cercant una paraula concreta i simplement eixa paraula no existeix. si mostrem un 404, deixem de mostrar a l'usuari la pàgina del cercador. En realitat, això no deixa de ser més que una 'cerca'. És com si a les notícies, en cas de no trobar un resultat concret, mostràrem un 404 en lloc de la pàgina on es diu que no s'han trobat els resultats.

Sé que el cas no és el mateix. Però tal qual es planteja actualment el diccionari multilingüe, la cosa és comporta així. Vols un 404 igualment?